### PR TITLE
Refactored AboutActivity and layout file

### PR DESCRIPTION
**Pull request by 4lch4**
The activity now sends people to the correct Contributors page instead of ones limited to a specific period of time.

The layout is also much simpler now and no longer contains unnecessary elements, such as the previous LinearLayout that wrapped the ScrollLayout and was completely useless.

This PR is quite a small one so I didn't include too much information. If you want me to explain some more I'd be glad to, just let me know.

### CODE REVIEW

**Comment by 4lch4** [comment]
Forgot to mention I also added some default dimensions for various things such as margins, card view elevation, etc., that I'll be using later on with more refactors.

In regards to the **CodeClimate**/**Codacy** issue, there's no need for a `default` statement as there are only ever two instances when that function is called and it's when a user is pressing on one of the views.

File: app/src/main/java/in/tosc/digitaloceanapp/activities/AboutActivity.java [hunk 1]
```diff
@@ -2,51 +2,48 @@
...
+     * Called by the {@link View} objects in the About Activity when they're pressed/clicked and
+     * will launch the default browser to either the GitHub repo for the app or the Contributors
+     * page for the repo depending on which button was pressed.
+     *
+     * @param view View object that was pressed
+     */
+    public void openPage(View view) {
+        Intent i = new Intent(Intent.ACTION_VIEW);
+
+        switch (view.getId()) {
```
[the remark this answers is not in the document]
Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/AboutActivity.java:36]: If we insist on fixing this issue, we can go with something like this:

```java
switch (view.getId()) {
    case R.id.relative_layout_github:
        // Send the user to the GitHub repo
        i.setData(Uri.parse("https://github.com/coding-blocks/DigitalOceanApp"));
        break;

    case R.id.relative_layout_see_all_contributors:
        // Send the user to the GitHub Contributors page
        i.setData(Uri.parse("https://github.com/coding-blocks/DigitalOceanApp/graphs/contributors"));
        break;

    default:
        return;
}
```

It fixes the issue, doesn't introduce any possible errors, and is simple to read.

**Comment by championswimmer** [comment]
Travis is still failing

**Comment by 4lch4** [comment]
Alright, sorry for the delay, I had a few other things come up that took my time. I'll take a look at this now and see if I can't get it fixed tonight.

**Comment by 4lch4** [comment]
I've been unable to figure out why it's crashing on Travis... That said, there have already been some changes so I'll close this PR and submit changes at a later date.

**Pull request closed without merging**